models/news.js

```diff
@@ -3,3 +3,22 @@ const db = require("../db/connection");
 exports.selectTopics = () => {
   return db.query("SELECT * from topics;").then((result) => result.rows);
 };
+
+exports.selectArticleById = (article_id) => {
+  return db
+    .query(
+      "select users.username AS author, title, article_id, body, topic, created_at, votes from articles INNER JOIN users ON articles.author = users.username where article_id = $1;",
+      [article_id]
+    )
+    .then((result) => {
+      const article = result.rows[0];
+      console.log(article);
```

Comment:
get rid of this console.log if it isn't necessary anymore

Reply:
removed